Fix UI window sizing (#247)
AY1920S1-CS2103-T16-3/main: docs/UserGuide.adoc

.  Refer to <<Features>> for details of each command.

> Recommended to be at a resolution of 1920x1080 with 100% (no) scaling for best usage. Weme requires a resolution that provides at least 750 pixels in height and 1000 pixels in width for smooth usage.

Reviewer: ```suggestion
We recommend a minimum resolution of 1920x1080 with 100% (no) scaling for best usage. Weme requires a resolution that provides at least 750 pixels in height and 1000 pixels in width for smooth usage.
```

Reviewer: Put this in a box too.
